com.playeveryware.eos/Runtime/Core/Config/SandboxId.cs

```diff
@@ -75,5 +75,14 @@ public override readonly int GetHashCode()
         {
             return (_value?.GetHashCode()) ?? 0;
         }
+
+        public override readonly string ToString()
+        {
+            // SandboxId is _sometimes_, but _not always_ a GUID value. In case
+            // it was entered into the configuration with a format that includes
+            // dashes and upper case letters, this function removes the former
+            // and changes the latter.
+            return _value.Replace("-", "").ToLower();
```

Comment:
How do you feel about setting _value to this (Replacing - and setting to Lower) when the value is set, rather than doing that work here?

Reply:
The reason I don't do that _here_ is because it would be very confusing to have the `ToString()` method actually alter the value. 

I _can_ however see the value in perhaps using it whenever `_value` _is_ being set (like in the `set` section of the Property).

Reply:
Actually, I realize that this implementation is incorrect - because valid sandbox ids can start with a `p-`, and this function would mess that up. I've updated the implementation to consider it more properly.